### ISSUE TITLE
Accessibility Testing for [vfs-mhv-secure-messaging, mhv-secure-messaging, Oracle Health curated list]

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#119644]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### VFS team name

vfs-mhv-secure-messaging

### Product name

mhv-secure-messaging

### Feature name

Oracle Health curated list

### GitHub label for team

mhv-va.gov-secure-messaging

### GitHub label for product

mhv-secure-messaging

### Public DSVA Slack channel for VFS team

mhv-secure-messaging

### Who completed the use of color and color contrast test?

- @steveg-IAT 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

- No color or contrast issues noticed

### Who completed the axe scans?

- @steveg-IAT 

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

- All Axe scans passed (issue in Wave scan detailed below)

### How did axe testing go?

- Passed

### Who completed the content zoom and reflow test?

- @steveg-IAT 

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [x] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

- Failed @ 400%, See details in commnet below

### Who completed the keyboard navigation test?

- @steveg-IAT 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

- No Keybd nav issues noticed

### Do you have any other results to share?

- see details in comments below
- new issues created:
  - https://github.com/department-of-veterans-affairs/va.gov-team/issues/119851  
  - https://github.com/department-of-veterans-affairs/va.gov-team/issues/119858

**Comment by steveg-IAT** [comment]
**Broken ARIA reference:** <img width="1269" height="810" alt="Image" src="https://github.com/user-attachments/assets/fb5d7a1d-f053-4903-8ac0-39c0eb89c000" />
